tests/exception-policy-ids-midstream-disabled/exception-policy-midstream-disabled-drop-packet/README.md

```diff
@@ -0,0 +1,15 @@
+# Test
+
+Check that the midstream exception policy is properly applied in case Suricata
+has stream midstream pick-up sessions disabled. In this test the exception policy
+for midstream sessions is set to ``drop-packet``. This test is for IDS mode.
+
+# Behavior
+
+We expect Suri to error out without starting as ``drop-packet`` isn't a valid
+exception policy value for the midstream exception policy.
+
```

Comment:
We must decide the behavior for ``drop-packet`` and ``drop-flow``, as the current one is different from this approach.